pkg/test/step.go

```diff
@@ -196,6 +198,16 @@ func (s *Step) Create(namespace string) []error {
 		if updated, err := testutils.CreateOrUpdate(ctx, cl, obj, true); err != nil {
 			errors = append(errors, err)
 		} else {
+			// if the object was created, register cleanup
+			if !updated && !s.SkipDelete {
+				obj := obj
+				// TODO(eddycharly): create context
```

Comment:
Please, remove this `TODO` and create an issue instead

Reply:
https://github.com/kudobuilder/kuttl/issues/420

Reply:
I removed the `TODO`